CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#### Unreleased
+   * Strings: Rename `flip` to `reverse`. `flip` is deprecated and will be
+     removed in one release.
```

Comment:
Also, this might technically be a breaking change since we could create conflicts with `reverse`s from other libraries.